browser/ntp_sponsored_images/view_counter_model.cc

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2020 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "brave/browser/ntp_sponsored_images/view_counter_model.h"
+
+#include "base/logging.h"
+
+namespace {
+
+constexpr int kInitialCountToBrandedWallpaper = 1;
+constexpr int kRegularCountToBrandedWallpaper = 3;
```

Comment:
nit - drop `Regular`

Reply:
also seems like this would make more sense as a pref?

Reply:
I don't believe this would make sense as a pref as it is a constant value. It is the constant gap between sponsored wallpapers. In other words there are usually (hence "Regular") 3 non-sponsored wallpapers in-between sponsored wallpapers. However, at startup (hence "Initial") there is only 1 non-sponsored wallpaper before getting a sponsored wallpaper.

Reply:
the reason to make it a pref would be to make it adjustable with settings, but maybe we don't care about that